syntax = "proto3";

package Packet;

message PlayerMovePacket {
	sint32 x = 1;
	sint32 y = 2;
	sint32 player_id = 3;
	bool relative = 4;
}